source/agora/test/ValidatorRecurringEnrollment.d

```diff
@@ -265,7 +265,7 @@ unittest
     // Let the last node catch up
     network.expectHeight([ GenesisValidators - 1 ], Height(GenesisValidatorCycle));
 
-    network.generateBlocks(iota(GenesisValidators),
+    network.generateBlocks(iota(GenesisValidators - 1),
```

Comment:
Why this change ?

Reply:
The last validator was put to sleep and just woke up but I think it maybe doesn't get the tx due to being banned.

Reply:
Validators should never be banned though

Reply:
It failed on new check for `hasTransactionHash` I added to `addBlock`.

Reply:
I'll debug the test and check. Maybe it is a bug.

Reply:
I increased the retry timeout to 4 seconds and it now passes.